view/frontend/web/js/view/payment/method-renderer/omise-offline-tesco-method.js

```diff
@@ -39,7 +41,7 @@ define(
              * @return {boolean}
              */
             isActive: function() {
-                return this.getOrderCurrency().toLowerCase() === 'thb' && this.getStoreCurrency().toLowerCase() === 'thb';
+                return paymentTools.getOrderCurrency().toLowerCase() === 'thb' && paymentTools.getStoreCurrency().toLowerCase() === 'thb';
```

Comment:
Are these currencies always used in lowercase? Would it be better to move the case transformation to the currency functions themselves?